install_dragonchain_uvn.sh

```diff
@@ -4,10 +4,8 @@
 ## Run on Ubuntu 18.04 LTS from AWS (probably will work on others but may be missing )
 
 # Variables
-DRAGONCHAIN_VERSION="4.0.0" 
-DRAGONCHAIN_HELM_CHART_FILENAME="dragonchain-k8s-1.0.0.tgz"
-DRAGONCHAIN_HELM_CHART_URL="https://dragonchain-core-docs.dragonchain.com/4.0.0/_downloads/36f9cc0584bac8b0a06f434d60e4c811/$DRAGONCHAIN_HELM_CHART_FILENAME"
-DRAGONCHAIN_HELM_VALUES_URL="https://dragonchain-core-docs.dragonchain.com/4.0.0/_downloads/f55b95b91076947c217a57e6b413c9c5/opensource-config.yaml"
+DRAGONCHAIN_VERSION="4.1.0" 
+DRAGONCHAIN_HELM_CHART_VERSION="1.0.2"
 
 REQUIRED_COMMANDS="sudo ls grep chmod tee sed touch cd timeout ufw savelog"
```

Comment:
We should add wget and curl to required commands since we use them. Pretty sure wget is in most images but I'm not sure about curl.

Reply:
Done